vnext/src/RNTester/js/examples-win/Text/TextExample.windows.tsx

```diff
@@ -79,6 +79,39 @@ export class TextExample extends React.Component<{}> {
 
     return (
       <RNTesterPage>
+        <RNTesterBlock title="textTransform">
```

Comment:
Looks like this is from the Android TextExample page.

The examples-win directory was mostly meant for custom windows examples, but TextExample was a bit special since long ago it converted from Flow to Typescript, forked, and stopped being kept up to date. There we're a number of files like this that have been kind of corroded by time before we had tooling to automate three-way-merge of our forked changes.

Consider removing this fork of TextExample and adding back the Flow-based Android one (with any needed patching) to js/examples. That way it will be kept up to date automatically. ACoates and I have been opoortunistically killing the Typescript forks of Flow files when we come across them, and have often found gaps when we do.

Reply:
Filed #5678